server/handlers.go

```diff
@@ -3,13 +3,50 @@ package server
 import (
 	"fmt"
 	"github.com/gin-gonic/gin"
+	"github.com/tempo-nksn/Tempo-Backend/models"
+	"math/rand"
+	"net/url"
+	"strconv"
 )
 
 func hello(c *gin.Context) {
 	c.String(200, "Hello User, your taxi is booked")
 }
 
 func dummyPost(c *gin.Context) {
+	var str struct {
+		Name string `json:"name"`
+	}
+	c.BindJSON(&str)
 	fmt.Printf("I am posting to ios")
-	c.JSON(200, "here is your post")
+	c.JSON(200, str)
 }
+
+// getTaxi calculates 3 random taxi location neat the user
+// In future it will respond with the location of the nearby taxis
```

Comment:
Write how you generate random location in a brief way

Reply:
Done